.github/workflows/ci.yml

```diff
@@ -14,7 +14,13 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [quality, docs,  django32-drf312, django32-drflatest]
+        toxenv: [
+          quality,
+          docs,
+          django32-drf314,
```

Comment:
`drf-314` is the `drf-latest` so we can remove this one as it'll be duplicate with the below envs.
```suggestion
```